CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unversioned
 
+- Minor: Added `is:<flags>` search filter to find messages of specific types. (#2653, #2671)
```

Comment:
I'll now try to make a wiki page about already existing search predicates. After that's done, you should probably make a PR to chatterino/wiki with a list of the flags that you can use with `"is"` and append link to the wiki page with those to the changelog entry.

Reply:
Chatterino/wiki#65 has now been merged in, you can try appending new stuff to the list of Search filters and maybe add some extra examples at the end in the same format I made those. As for version note `(added in 2.X.X)` just say `(added in nightly build)` or just nightly.